rotkehlchen/db/upgrades/v36_v37.py

```diff
@@ -85,31 +117,31 @@ def _update_history_events_schema(write_cursor: 'DBCursor', conn: 'DBConnection'
         read_cursor.execute('SELECT * from history_events')
         for entry in read_cursor:
             location = Location.deserialize_from_db(entry[4])
-            if isinstance(entry[1], str):
-                event_identifier = entry[1]
-            else:  # is an EVM transaction event
-                if location not in EVM_LOCATIONS:
-                    # data is somehow wrong. Let's try to fix location
-                    chain_id = write_cursor.execute('SELECT chain_id from evm_transactions WHERE tx_hash=?', (entry[1],)).fetchone()  # noqa: E501
-                    if chain_id is not None:
-                        location = Location.from_chain_id(chain_id)
-                    else:  # could not find tx_hash. Let's just default to ethereum
-                        location = Location.ETHEREUM
-
-                event_identifier = f'{location.to_chain_id()}{deserialize_evm_tx_hash(entry[1]).hex()}'  # noqa: E501 # pylint: disable=no-member
+            db_event_identifier = entry[1]
+            if location in EVM_LOCATIONS:
+                event_identifier = f'{location.to_chain_id()}{deserialize_evm_tx_hash(db_event_identifier).hex()}'  # noqa: E501 # pylint: disable=no-member
+            elif location == Location.KRAKEN or db_event_identifier.startswith(b'rotki_events'):
+                # kraken is the only location with basic history event entry type that doesn't
+                # start with 'rotki_events'
+                event_identifier = db_event_identifier.decode()
+            else:
+                # this shouldn't happen, leaving it here to avoid failing during the upgrade
+                # _move_event_locations should have moved already all the events that are related
+                # to evm transactions with the wrong location
+                log.critical(f'Unexpected event found with {db_event_identifier=} and {location=}. {entry=} Skipping')  # noqa: E501
```

Comment:
The other fields are already part of `entry`.

```suggestion
                log.critical(f'Unexpected event {entry=} found Skipping')  # noqa: E501
```

So you think we should just remove the unexpected event instead of setting some default location?